code/modules/atmospherics/machinery/components/components_base.dm

```diff
@@ -114,35 +114,6 @@
 		to_return += parents[i]
 	return to_return
 
-/**
- * Called by nullify_node(), used to remove the pipeline the component is attached to
- * Arguments:
- * * -reference: the pipeline the component is attached to
- */
-/obj/machinery/atmospherics/components/proc/nullify_pipenet(datum/pipeline/reference)
-	if(!reference)
```

Comment:
Why move this down exactly?

Reply:
Eh moved them when copied it back, no real reason tbh, do you want me to change it back?

Reply:
This doesn't really matter, mostly just confused me